app/assets/stylesheets/custom.css.scss

```diff
@@ -1,3 +1,15 @@
+.logo {
+  text-transform: uppercase;

```

Comment:
Properties should be ordered color, font-family, font-size, font-weight, text-transform